_posts/0000-01-04-carpnick.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Caption here - **carpnick** was here
```

Comment:
@carpnick this is a great caption, but I don't think it has anything to do with dojo's. Do you think we should update?